docker/README.md

```diff
@@ -327,6 +327,28 @@ You should now be able to configure [Jetpack Backup & Scan](https://jetpack.com/
 - Server password: `wordpress`
 - WordPress installation path: `/var/www/html`
 
+## Custom plugins & themes in the container
+
+Jetpack Docker environment can be wonderful for developing your own plugins and themes, too.
+
+Since everything under `mu-plugins` and `wordpress/wp-content` are git-ignored, you'll want to keep those folders outside Jetpack repository folder and link them as volumes to your Docker instance.
+
+1. First ensure your containers are stopped (`yarn docker:stop`).
+1. Create a docker-compose file. You can place it anywhere in your computer:
+	```yml
+	version: '3.3'
+	services:
+	  wordpress:
+	    volumes:
+	      - ~/my-plugin:/var/www/html/wp-content/plugins/my-plugin
```

Comment:
Is `~/my-plugin` where my own plugin would be developed on my machine, and `/var/www/html/wp-content/plugins/my-plugin` where it would be linked?

Reply:
Yep! Do you recon I should clarify that more? I expected it to be clear from path names but I know our Docker setup inside-out so I'm blind to things like these.

Reply:
Elaborated more in https://github.com/Automattic/jetpack/pull/12189/commits/52b18dbcd2ffe69e07d67b82a308bb4e3f03aee1

Reply:
Thank you!